tests/README.md

```diff
@@ -52,21 +58,21 @@ Local Testing
 From the jetty-runtime/tests directory:
 
 ```
-> mvn install -Plocal -Djetty.test.image=jetty:9.4
+> mvn install
 ```
 
-This will activate the 'local' profile and enable testing.  The tests activated under this profile will make use of the locally installed image and tag referenced in the jetty.test.image property.  The spotify docker-maven-plugin is used to build the test docker container and the io.fabric8 docker plugin is used to manage the integration test lifecycle. Local tests may have a smaller scope as they are not intended to the complete Google Flex environment.  Local testing is intended to test and validate configuration of Jetty and basic environment. 
+The tests activated under this profile will make use of the locally installed image and tag referenced in the jetty.test.image property (default 'jetty:${docker.tag.short}').  The spotify docker-maven-plugin is used to build the test docker container and the io.fabric8 docker plugin is used to manage the integration test lifecycle. Local tests may have a smaller scope as they are not intended to the complete Google Flexible environment.  Local testing is intended to test and validate configuration of Jetty and basic environment. 
 
 Remote Testing
 =====
 
 Again from the jetty-runtime/tests directory:
 
 ```
-> mvn install -Premote -Djetty.test.image=gcr.io/{project}/jetty9:9.4
+> mvn install -Ptest.remote -Djetty.test.image=gcr.io/{project}/jetty9:9.4
```

Comment:
In this repo, if you do a build, you should see `jetty:9.4` generated under `docker images`.